src/lib/hooks/use-best-quote/utils/flashmint.ts

```diff
@@ -49,18 +49,15 @@ async function getEnhancedFlashMintQuote(
       .length > 0
   if (!isAllowedCurrency) return null
   try {
-    // Create an instance of ZeroExApi (to pass to quote functions)
-    const networkKey = getNetworkKey(chainId)
-    const swapPathOverride = `/${networkKey}/swap/v1/quote`
-    const zeroExApi = getConfiguredZeroExApi(swapPathOverride)
+    const swapQuoteProvider = getConfiguredZeroExSwapQuoteProvider(chainId)
     const request = {
       isMinting,
       inputToken: { ...inputToken, address: inputTokenAddress },
       outputToken: { ...outputToken, address: outputTokenAddress },
       indexTokenAmount,
       slippage,
     }
-    const quoteProvider = new FlashMintQuoteProvider(jsonRpcProvider, zeroExApi)
+    const quoteProvider = new FlashMintQuoteProvider(rpcUrl, swapQuoteProvider)
```

Comment:
Most changes are just satisfy the new constructor requirement for the `FlashMintQuoteProvider`